src/main/resources/META-INF/accesstransformer.cfg

```diff
@@ -87,4 +87,6 @@ public net.minecraft.world.level.block.state.BlockBehaviour$Properties f_60886_
 public net.minecraft.world.level.chunk.ChunkGenerator f_223020_ #featuresPerStep
 public net.minecraft.world.level.levelgen.placement.PlacedFeature m_226368_(Lnet/minecraft/world/level/levelgen/placement/PlacementContext;Lnet/minecraft/util/RandomSource;Lnet/minecraft/core/BlockPos;)Z # placeWithContext
 
-public net.minecraft.world.phys.AABB m_82325_(Lnet/minecraft/world/phys/AABB;Lnet/minecraft/world/phys/Vec3;[DLnet/minecraft/core/Direction;DDD)Lnet/minecraft/core/Direction; # getDirection
\ No newline at end of file
+public net.minecraft.world.phys.AABB m_82325_(Lnet/minecraft/world/phys/AABB;Lnet/minecraft/world/phys/Vec3;[DLnet/minecraft/core/Direction;DDD)Lnet/minecraft/core/Direction; # getDirection
+
+public net.minecraft.client.renderer.entity.ItemRenderer m_115152_(Lcom/mojang/blaze3d/vertex/BufferBuilder;IIIIIIII)V # fillRect
```

Comment:
We sort our ATs alphabetically but honestly I feel like with how simple a method it is we may want to just copy it to `GuiUtils`, or better yet (unless it is because of the set gl states) is there a reason this can't just use `GuiUtils#fill` and pass it a `new PoseStack()` as the current pose?